src/utility.jl

```diff
@@ -0,0 +1,80 @@
+
+# TODO: 
+# - tidy up this file
+
+
+Base.zero(lv::LorentzVector{CS,T}) where {CS,T} = LorentzVector{CS}(ntuple(i->zero(T),4)...)
+
+
+Base.:(==)(lv1::LorentzVector, lv2::LorentzVector) = false
+
+function Base.:(==)(lv1::LorentzVector{CS}, lv2::LorentzVector{CS}) where {CS<:LorentzVectorBase.AbstractCoordinateSystem}
+    return lv1.data==lv2.data
+end
+
+Base.isapprox(lv1::LorentzVector,lv2::LorentzVector; args...) = false
+
+function Base.isapprox(
+    lv1::LorentzVector{CS}, 
+    lv2::LorentzVector{CS}; 
+    atol::Real=0, 
+    rtol::Real=atol>0 ? 0 : sqrt(eps())
+) where {CS<:LorentzVectorBase.AbstractCoordinateSystem}
+    all(isapprox.(lv1.data,lv2.data;atol,rtol))
+end
+
+
+# phi is in [0,2pi) by definition in LorentzVectorBase
+function phi02pi(lv::LorentzVector)
+#    return mod2pi(LorentzVectorBase.phi(lv))
+    return phi(lv) < 0.0 ? phi(lv) + 2π : phi(lv)
+end
+
+function phi_mpi_pi(x)
+    twopi = 2pi
+    while (x >= pi)
+        x -= twopi
```

Comment:
hello `twopi` 🙂 I think the compiler is easy with `2π` but I might be wrong. Just nit-picking...